frontend/Dockerstage

```diff
@@ -3,4 +3,4 @@ FROM node:latest AS svelte
 COPY --chown=node:node $PACKAGE_DIR /home/node/frontend
 
 WORKDIR /home/node/frontend
-RUN su node -c "npm install && npm run build"
+RUN su node -c "npm ci && npm run build"
```

Comment:
Using this (and a `package-lock.json`) makes sense to me in general, but I don't know if it makes sense to use here specifically. 

This is used not just for CI, but also for building regular Docker images. I can imagine a world in which someone building a Docker image that uses the front end doesn't want the "fail fast" approach of the `npm ci` command and would instead prefer `npm install`. 

Same comment applies to the other change in this PR to switch `npm install` to `npm ci` in the `ofrak_core/Makefile`.

Reply:
I agree that it should cause an error in CI and before code is merged. But it seems overly strict to impose on local development. 

In my estimation, we are more likely to get a bug report from someone who cannot build their image due to hard failures than we are from someone who has GUI bugs due to minor dependency version mismatches. 

Our main users are security researchers, not software engineers. Sometimes, allowing them to play fast and loose during development is critical for reducing friction when using OFRAK (which is already higher than I would like). 

I still think we should not use `npm ci` everywhere. We should use it only in CI.